docs(radio): edit radio & radio group documentation (#1905)
IgniteUI/igniteui-docfx: en/components/radio_button.md

<igx-radio [(ngModel)]="selected" value="option2">Option 2</igx-radio>
```

#### Label

> The `labelPosition` property can be used for changing the default position of the label in a radio component. Users can choose between `before` and `after` options. If not specified, the label will be placed after the radio button.

Reviewer: ```suggestion
The `labelPosition` property can be used to change the default position of the label in the radio component. Users can choose between `before` and `after. If not specified, the label will be placed after the radio button.
```
